core/templates/dev/head/css/oppia.css

```diff
@@ -3831,15 +3831,6 @@ span.oppia-issues-learner-action {
   margin-bottom: 2em;
 }
 
-.oppia-issues-hexagon-text {
```

Comment:
Is this not used anywhere? Grepping suggests that it arises in core/templates/dev/head/pages/exploration_editor/statistics_tab/early_quit_issue_directive.html and some other files. (Or are those html files supposed to be deleted?)

Reply:
Whoops, reverted.